Cargo.toml

```diff
@@ -1,8 +1,16 @@
 [package]
-name = "crate-template"
+name = "vm-superio"
 version = "0.1.0"
-authors = [TODO]
+description = "Emulation for legacy devices"
+keywords = ["legacy"]
```

Comment:
I would add "emulation", "serial" (or "uart") as keywords as well.

Reply:
I'll update in a future PR if there are no other comments.